Terminal.Gui/ConsoleDrivers/CursesDriver/UnmanagedLibrary.cs

```diff
@@ -76,9 +74,9 @@ static UnmanagedLibrary ()
         IsMacOSPlatform = platform == PlatformID.Unix && GetUname () == "Darwin";
         IsLinux = platform == PlatformID.Unix && !IsMacOSPlatform;
 
-        IsWindows = (platform == PlatformID.Win32NT)
-                    || (platform == PlatformID.Win32S)
-                    || (platform == PlatformID.Win32Windows);
+        IsWindows = platform == PlatformID.Win32NT
```

Comment:
.net 8 made some improvements to this and has some new types for more comprehensive platform identification than before. Might be worth checking out later on.